common/predictive-text/build.sh

```diff
@@ -25,80 +25,51 @@ cd "$(dirname "$THIS_SCRIPT")"
 builder_check_color "$@"
 
 builder_describe "Builds the lm-layer module" \
+  "@../web/keyman-version" \
+  "@../web/lm-worker" \
```

Comment:
```bash
  "@../models/templates" \
  "@../models/types" \
  "@../models/wordbreakers"
```

While linked in via the whole `tsc -b` mechanism, they are dependencies for this module.

Reply:
There are other places where that's been commented as notes for the future, but not here.  Maybe leave a similar note?

Reply:
Done. Are you able to approve this PR? :grin: